packages/adapter-prisma/lib/adapter-prisma.js

```diff
@@ -312,14 +312,39 @@ class PrismaListAdapter extends BaseListAdapter {
     return Object.keys(include).length > 0 ? include : undefined;
   }
 
+  _formatValue(value, path) {
+    let format = this._getFieldStorageFormat(path);
+    switch (format) {
+      case 'Boolean':
+        return Boolean(value);
+      case 'DateTime':
+      case 'String':
+        return value + '' || '';
+      case 'Int':
+      case 'Float':
+      case 'Decimal':
+        return Number(value) || null;
```

Comment:
should this change from Number to parseInt, parseFloat etc. also does this decimal represented as string in database or number?

Reply:
Fixed, thanks!